install/openebs-operator.yaml

```diff
@@ -0,0 +1,2342 @@
+              privileged: true
+            volumeMounts:
+            - name: sockfile
+              mountPath: /var/run
+            - name: conf
+              mountPath: /usr/local/etc/istgt
+            - name: dummyfile
+              mountPath: /tmp/cstor
+          {{- if eq $isMonitor "true" }}
+          - image: {{ .Config.VolumeExporterImage.value }}
+            name: maya-volume-exporter
+            args:
+            - "-e=cstor"
+            command: ["maya-exporter"]
+            ports:
+            - containerPort: 9500
+              protocol: TCP
+            volumeMounts:
+            - name: sockfile
+              mountPath: /configs
+          {{- end}}
+          - name: cstor-volume-mgmt
+            image: {{ .Config.VolumeControllerImage.value }}
+            imagePullPolicy: Always
+            ports:
+            - containerPort: 80
+            env:
+            - name: OPENEBS_IO_CSTOR_VOLUME_ID
+              value: {{ .TaskResult.cvolcreateputvolume.cstorid }}
+            securityContext:
+              privileged: true
+            volumeMounts:
+            - name: sockfile
+              mountPath: /var/run
+            - name: conf
+              mountPath: /usr/local/etc/istgt
+            - name: dummyfile
+              mountPath: /tmp/cstor
+          volumes:
+          - name: sockfile
+            emptyDir: {}
+          - name: conf
+            emptyDir: {}
+          - name: dummyfile
+            emptyDir: {}
+---
+# runTask to create cStorVolumeReplica/(s)
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cstor-volume-create-put-replica-0.7.0
+  namespace: openebs
+data:
+  meta: |
+    apiVersion: openebs.io/v1alpha1
+    runNameSpace: openebs
+    kind: CStorVolumeReplica
+    action: put
+    id: cstorvolumecreatereplica
+    {{/*
+    Fetch all the cStorPool uids into a list.
+    Calculate the replica count
+    Add as many poolUid to resources as there is replica count
+    */}}
+    {{- $poolUids := keys .ListItems.cvolPoolList.pools }}
+    {{- $replicaCount := int64 .Config.ReplicaCount.value }}
+    repeatWith:
+      resources:
+      {{- range $k, $v := $poolUids }}
+      {{- if lt $k $replicaCount }}
+      - {{ $v }}
+      {{- end }}
+      {{- end }}
+  task: |
+    kind: CStorVolumeReplica
+    apiVersion: openebs.io/v1alpha1
+    metadata:
+      {{/*
+      We pluck the cStorPool name from the map[uid]name:
+      { "uid1":"name1","uid2":"name2","uid2":"name2" }
+      The .ListItems.currentRepeatResource gives us the uid of one
+      of the pools from resources list
+      */}}
+      name: {{ .Volume.owner }}-{{ pluck .ListItems.currentRepeatResource .ListItems.cvolPoolList.pools | first }}
+      labels:
+        cstorpool.openebs.io/name: {{ pluck .ListItems.currentRepeatResource .ListItems.cvolPoolList.pools | first }}
+        cstorpool.openebs.io/uid: {{ .ListItems.currentRepeatResource }}
+        cstorvolume.openebs.io/uid: {{ .Volume.owner }}
```

Comment:
We should update this and ask @gkGaneshR to make corresponsing changes in his code. This label name is incorrect.

Reply:
Do you mean to replace cstorvolumeCR.uid with cstorvolumeCR.Name ? If so, It is just a string change and no code change is required from my end.

Reply:
Yes.

Reply:
it is only a string change for me, both will work fine in cstor-pool-mgmt container. No change is required from my end. Please discuss with @avishnu to confirm which one to use, coz. this needs to be common between control plane and data plane.

Reply:
`cstorvolume.openebs.io/uid` -> `cstorvolume.openebs.io/name`